samples/kaspresso-sample/src/androidTest/kotlin/com/kaspersky/kaspressample/screen/ScrollViewWithPaddingScreen.kt

```diff
@@ -0,0 +1,18 @@
+package com.kaspersky.kaspressample.screen
+
+import com.kaspersky.kaspressample.R
+import com.kaspersky.kaspressample.autoscrollfallback.AutoscrollFallbackActivity
+import com.kaspersky.kaspresso.screens.KScreen
+import io.github.kakaocup.kakao.text.KButton
+
+object ScrollViewWithPaddingScreen : KScreen<ScrollViewWithPaddingScreen>() {
+
+    override val layoutId: Int = R.layout.activity_scrollview_with_padding
+    override val viewClass: Class<*> = AutoscrollFallbackActivity::class.java
+
+    val hbutton1 = KButton { withId(R.id.hvText1) }
+    val hbutton7 = KButton { withId(R.id.hvText7) }
+    val button1 = KButton { withId(R.id.tvText1) }
+    val button20 = KButton { withId(R.id.tvText20) }
+    val nnhbutton5 = KButton { withId(R.id.nnhtvText5) }
```

Comment:
Just a nitpick, I think `camelCase` would be better here :)

Reply:
- [ ] camelCase